internal/pkg/deploy/cloudformation/env.go

```diff
@@ -65,3 +66,16 @@ func (cf CloudFormation) streamEnvironmentResponse(done chan struct{}, resp chan
 		Err: err,
 	}
 }
+
+// EnvStack returns the CloudFormation stack of the environment.
+func (cf CloudFormation) EnvStack(projectName, envName string) (*sdkcloudformation.Stack, error) {
+	conf := stack.NewEnvStackConfig(&deploy.CreateEnvironmentInput{
+		Project: projectName,
+		Name:    envName,
+	})
+	descr, err := cf.cfnClient.Describe(conf.StackName())
+	if err != nil {
+		return nil, err
+	}
+	return descr.SDK(), nil
+}
```

Comment:
What do you think of changing the follow suggestion instead?

```suggestion
// GetEnvironment returns the Environment metadata from the CloudFormation stack.
func (cf CloudFormation) GetEnvironment(projectName, envName string) (*archer.Environment, error) {
	conf := stack.NewEnvStackConfig(&deploy.CreateEnvironmentInput{
		Project: projectName,
		Name:    envName,
	})
	descr, err := cf.cfnClient.Describe(conf.StackName())
	if err != nil {
		return nil, err
	}
	return conf.ToEnv(descr.SDK())
}
```

Otherwise, now the cli needs to do this work and take an extra dependency on `deploy/cloudformation/stack`.